src/Elmish.WPF/ViewModel.fs

```diff
@@ -196,21 +187,22 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     | CmdData d ->
         let exec = measure name "exec" d.Exec
         let canExec = measure name "canExec" d.CanExec
-        let execute _ = exec currentModel |> ValueOption.iter dispatch
-        let canExecute _ = canExec currentModel
+        let execute _ = currentModel |> exec |> ValueOption.iter dispatch
+        let canExecute _ = currentModel |> canExec
         Cmd (Command(execute, canExecute, false), canExec)
     | CmdParamData d ->
         let exec = measure2 name "exec" d.Exec
         let canExec = measure2 name "canExec" d.CanExec
-        let execute param = exec param currentModel |> ValueOption.iter dispatch
```

Comment:
TBH I like this better as it is. I don't see a need to use pipelines for multi-parameter functions unless it actually makes things more clear, and in this case, `param` and `currentModel` are both equally "important" parameters.

For consistency, I'd then also like to keep `(can)exec currentModel` above.

Reply:
Ok, reverted.